test-helpers/src/hardhat.ts

```diff
@@ -0,0 +1,28 @@
+import { JsonRpcServer } from 'hardhat/internal/hardhat-network/jsonrpc/server'
+import { TASK_NODE_GET_PROVIDER, TASK_NODE_CREATE_SERVER } from 'hardhat/builtin-tasks/task-names'
+import { run } from 'hardhat'
+
```

Comment:
You can export the private key from here

Reply:
Thanks @RodrigoAD, very good idea. I was trying to directly import from hardhat config and was running into path issues.